src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,55 @@
 import React from 'react';
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type PostProps = {
+  post: Post
+};
+
+export const PostInfo: React.FC<PostProps> = ({ post }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = post;
+
+  if (user === null) {
+    return null;
+  }
```

Comment:
just do not render `UserInfo`, everything else should be rendered
you can render `'Posted anonymously'`

Reply:
Done.